Assets/HoloToolkit/Input/Scripts/InputEventData/BaseInputEventData.cs

```diff
@@ -5,6 +5,21 @@
 
 namespace HoloToolkit.Unity.InputModule
 {
+    public enum MixedRealityInputType
```

Comment:
I think we could use a better name.  Mixed Reality is generally for VR/AR experiences, but we have touchscreen here?

Reply:
Agreed, and this enum may not even be necessary (see note on incorporating into IInputHandler).

There's a big push for Touchscreen in MR. Asymmetrical multiplayer, observers, etc. Haven't seen many examples but it's considered part of the ecosystem.

Reply:
Yeah I agree, we should add to the enum that's already in the `IInputHandler`

Reply:
I've added these to the InteractionSourceInfo.

This breaks that enum's parity with Unity's UnityEngine.XR.WSA.Input.InteractionSourceKind but there are no obvious consequences.

Reply:
I think it should be fine.  I don't see them changing that enum anytime soon anyway. (But then again it is Unity)